server/test/unit/com/thoughtworks/go/server/service/GoConfigServiceTest.java

```diff
@@ -1135,11 +1135,33 @@ public void shouldSayThatAUserIsAuthorizedToViewAndEditTemplatesWhenTheUserHasPe
 
         assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(templateAdminName)), is(true));
         assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(secondTemplateAdminName)), is(true));
-        assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(new CaseInsensitiveString(theSuperAdmin))), is(false));
+        assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(new CaseInsensitiveString(theSuperAdmin))), is(true));
```

Comment:
No impact at all. Because the method `isUserAuthorizedToViewAndEditTemplates` was only checking the template authorization and not checking if user is a super admin. So, wherever we used this method on the rails views, the `is_user_an_admin` method was called. Like this - 
`is_user_an_admin? || is_user_authorized_to_view_and_edit_templates?`